retailer-oas/schemas/Timeslots.yaml

```diff
@@ -0,0 +1,43 @@
+properties:
+  data:
+    type: object
+    properties:
+      pickingLocationRef:
+        description: Picking Location Reference
+        format: uuid
+        example: "7996"
+        type: string
+      serviceType:
+        type: object
+        properties:
+          pickDelivery:
+            type: boolean
+          pickCollect:
+            type: boolean
+        description: Services allowed in this Picking Location.
+      timeslots:
+        type: array
+        items:
+          allOf:
+            - $ref: "./Timeslot.yaml"
+            - type: object
+              properties:
+                capacity:
+                  $ref: "./Capacity.yaml"
+                availability:
+                  type: boolean
+                  description: Indicates whether this Timeslot is available to be reserved or not.
+                  example: true
+  pagination:
+    type: object
+    properties:
+      limit:
```

Comment:
question: limit and page aren't query params?

Reply:
you're right. I will change the quantity param for the limit.
adding the page as well.